src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs

```diff
@@ -475,7 +495,7 @@ private void CheckStandardTransaction(MempoolValidationContext context)
             foreach (TxOut txout in tx.Outputs)
             {
                 ScriptTemplate script = this.network.StandardScriptsRegistry.GetTemplateFromScriptPubKey(txout.ScriptPubKey);
-                if (script == null) //!::IsStandard(txout.scriptPubKey, whichType, witnessEnabled))  https://github.com/bitcoin/bitcoin/blob/aa624b61c928295c27ffbb4d27be582f5aa31b56/src/policy/policy.cpp#L57-L80
+                if (script == null) //!::IsStandard(txout.scriptPubKey, whichType, witnessEnabled)) https://github.com/bitcoin/bitcoin/blob/aa624b61c928295c27ffbb4d27be582f5aa31b56/src/policy/policy.cpp#L57-L80
```

Comment:
What about this?

Reply:
Effectively this is an equivalent check - if the script registry does not see the output as a standard type, `script` will be `null`.

The registry includes the witness script types.